plugin/kubernetes/xfr.go

```diff
@@ -84,10 +84,6 @@ func (k *Kubernetes) Transfer(zone string, serial uint32) (<-chan []dns.RR, erro
 				endpointsList := k.APIConn.EpIndex(svc.Name + "." + svc.Namespace)
 
 				for _, ep := range endpointsList {
-					if ep.Name != svc.Name || ep.Namespace != svc.Namespace {
```

Comment:
why this change?

Reply:
I assume that this is because previously there was a 1:1 relation service - endpoint, referenced by name.
with endpoint slices service to slices is 1:N and the slices reference the service by label

Reply:
It was an pointless check (name and namespace would always match, since we are getting the list from an index lookup on name and namespace).
And now, comparing object.Endpoint.Name to object.Service.Name is no longer valid anyways, since the object.Endpoint could have been derived from an api.EndpointSlice instead of a api.Endpoint.

Reply:
ack ,thanks